test/integration/multi.yml

```diff
@@ -11,9 +11,7 @@
     es_data_dirs:
       - "/opt/elasticsearch/master"
     es_config:
-      discovery.zen.ping.unicast.hosts: "localhost:9300"
       http.port: 9200
-      transport.tcp.port: 9300
```

Comment:
I think this should still be left in as `transport.port` otherwise it goes against our recommendation of always setting it.

Reply:
well the goal was to avoid managing different config for 6.x and 7.x like that:
```

    es_config_6x:
      http.port: 9200
      transport.tcp.port: 9300
      node.data: false
      node.master: true
      bootstrap.memory_lock: tru
    es_config_7x:
      http.port: 9200
      transport.tcp.port: 9300
      node.data: false
      node.master: true
      bootstrap.memory_lock: tru
    es_config: "{{ es_config_7x if es_major_version == '7.x' else es_config_6x }}"
```

But we can add it